flink-runtime/src/main/java/org/apache/flink/runtime/security/modules/JaasModule.java

```diff
@@ -140,10 +155,12 @@ public DynamicConfiguration getCurrentConfiguration() {
 	/**
 	 * Generate the default JAAS config file.
 	 */
-	private static File generateDefaultConfigFile() {
+	private static File generateDefaultConfigFile(String workingDir) {
+		checkArgument(workingDir != null, "working directory should not be null.");
 		final File jaasConfFile;
 		try {
-			Path jaasConfPath = Files.createTempFile("jaas-", ".conf");
+			Path path = Paths.get(workingDir);
+			Path jaasConfPath = Files.createTempFile(path, "jaas-", ".conf");
```

Comment:
Why is it `jaas-` and not `jaas`?

Reply:
It names with "jaas-" when introduced this module, so I reuse "jaas-" not "jaas" in current pr.
I think the author wanted to add a "-" between "jaas" and the UUID.
I attached a picture from our production env
![image](https://user-images.githubusercontent.com/1425623/69237074-cf2a1980-0bcf-11ea-8572-b4bcb30f7228.png)

Reply:
Ah yes, I misunderstood what `createTempFile()` does. It adds the random number to the prefix. 👌